src/components/PopUpText.tsx

```diff
@@ -46,7 +46,7 @@ const PopUpText: React.FC = () => {
     <div className="orbit_title_popup_container">
       <div className="orbit_title_popup">
         <div className="orbit_title_info_header">{headerText}</div>
-        <div className="orbit_title_info_body">
+        <div className="orbit_title_info_body" tabIndex={0}>
```

Comment:
It was telling me that txt or body elements that require scrolling need to be keyboard accessible.